nameko/testing/services.py

```diff
@@ -201,6 +204,9 @@ def on_worker_teardown(worker_ctx):
                 yield waiter_result
 
 
+entrypoint_waiter.Timeout = type("Timeout", (Exception,), {})
```

Comment:
this seems needlessly fancy. if you really want you could prefix the class with an underscore when defining